frontend/src/components/ShootListRow.vue

```diff
@@ -6,119 +6,137 @@ SPDX-License-Identifier: Apache-2.0
-        </v-col>
-        <v-col class="shrink" >
-          <div class="d-flex flew-row" v-if="!isShootMarkedForDeletion">
-            <self-termination-warning :expirationTimestamp="shootExpirationTimestamp"></self-termination-warning>
-            <version-expiration-warning :shootItem="shootItem"></version-expiration-warning>
-            <hibernation-schedule-warning
-              v-if="isShootHasNoHibernationScheduleWarning"
-              :name="shootName"
-              :namespace="shootNamespace"
-              :purpose="shootPurpose">
-            </hibernation-schedule-warning>
-          </div>
-        </v-col>
-      </v-row>
-    </td>
-    <td class="nowrap" v-if="this.headerVisible['infrastructure']">
-      <vendor :shootItem="shootItem"></vendor>
-    </td>
-    <td class="nowrap" v-if="this.headerVisible['seed']">
-      <shoot-seed-name :shootItem="shootItem" />
-    </td>
-    <td class="nowrap" v-if="this.headerVisible['technicalId']">
-      <div class="d-flex align-center justify-start flex-nowrap fill-height">
-        <span>{{shootTechnicalId}}</span>
-        <copy-btn :clipboard-text="shootTechnicalId"></copy-btn>
-      </div>
-    </td>
-    <td class="nowrap" v-if="this.headerVisible['createdBy']">
-      <account-avatar :account-name="shootCreatedBy"></account-avatar>
-    </td>
-    <td class="nowrap" v-if="this.headerVisible['createdAt']">
-      <v-tooltip top>
-        <template v-slot:activator="{ on }">
-          <div v-on="on">
-            <time-string :date-time="shootCreationTimestamp" mode="past"></time-string>
-          </div>
-        </template>
-        {{ shootCreatedAt }}
-      </v-tooltip>
-    </td>
-    <td class="nowrap text-center" v-if="this.headerVisible['purpose']">
-      <purpose-tag :purpose="shootPurpose"></purpose-tag>
-    </td>
-    <td class="text-left nowrap" v-if="this.headerVisible['lastOperation']">
-      <div>
-        <shoot-status
-         :popperKey="`${shootNamespace}/${shootName}`"
-         :shootItem="shootItem">
-        </shoot-status>
-      </div>
-    </td>
-    <td class="nowrap text-center" v-if="this.headerVisible['k8sVersion']">
-      <shoot-version :shoot-item="shootItem" chip></shoot-version>
-    </td>
-    <td class="nowrap text-center" v-if="this.headerVisible['readiness']">
-      <status-tags :shootItem="shootItem"></status-tags>
-    </td>
-    <td v-if="this.headerVisible['accessRestrictions']">
-      <access-restriction-chips :selectedAccessRestrictions="shootSelectedAccessRestrictions"></access-restriction-chips>
-    </td>
-    <td class="nowrap" v-if="this.headerVisible['ticket']">
-      <v-tooltip top>
-        <template v-slot:activator="{ on }">
-          <div v-on="on">
+    <td v-for="cell in cells" :key="cell.header.value" :class="cell.header.class">
+      <template v-if="cell.header.value === 'project'">
+        <router-link class="cyan--text text--darken-2" :to="{ name: 'ShootList', params: { namespace: shootNamespace } }">
```

Comment:
Not part of this PR but I want to mention it. The name prefix `shoot` make things very complicated. I am not sure if this is really necessary.

Reply:
this is coming from the mixin, but we could introduce a `namespace` computed property

Reply:
we agreed to discuss this topic again together with @grolu